src/components/RandomMovieBackdrop/RandomMovieBackdrop.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+
+
+const RandomMovieBackdrop = (props) => {
+    const randomMovieIndex = Math.floor(Math.random() * props.movies.length)
+    const randomBackdrop = (props.movies[randomMovieIndex].backdrop_path)
+    const randomTitle = (props.movies[randomMovieIndex].title)
+
+    return (
+        <section className="random-movie-section">
+                <h2 className="backdrop-title"> {randomTitle} </h2>
```

Comment:
This is awesome - super straightforward. We could potentially add a button for the user to learn more details about the random movie, too!